HTML/task2/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel = "stylesheet" href = "style.css"/>    
+    <title>Task 2 - filter the colors</title>
+</head>
+<body>
+    <header><h1>Select colors</h1></header>
+    <main class = 'wrapper'>
```

Comment:
you don't need to use spaces in front and after `=` also generally there is a rule that in js we use single quotes and in html double, but at least keep consistency as you use both of them and it is not very good!!!